vendor-bin/phpunit/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require": {
+        "phpunit/phpunit": "^7.0"
```

Comment:
phpunit dependencies need to be compatible with the project dependencies because they run in the same PHP process when running tests. So isolating this dependency could actually break things.